notebooks/split_labels.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
<p>Would it make sense to store this in the repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>train object detection csv (tracked by dvc) </p><p>val object detection csv (tracked by dvc)</p><p> </p><ul><li>gs uri </li><li>bboxes </li><li>train/val id </li><li>attributes from datasets </li></ul>